AllEvents.TicketManagement/src/Core/AllEvents.TicketManagement.Application/Features/Events/Handlers/GetAllEventsQueryHandler.cs

```diff
@@ -7,35 +7,45 @@ namespace AllEvents.TicketManagement.Application.Features.Events.Handlers
 {
     public class GetAllEventsQueryHandler : IRequestHandler<GetAllEventsQuery, PagedResult<EventModel>>
     {
-        private readonly IEventRepository eventRepository;
+        private readonly IAllEventsDbContext _dbContext;
 
-        public GetAllEventsQueryHandler(IEventRepository eventRepository)
+        public GetAllEventsQueryHandler(IAllEventsDbContext dbContext)
         {
-            this.eventRepository = eventRepository;
+            _dbContext = dbContext;
         }
 
         public async Task<PagedResult<EventModel>> Handle(GetAllEventsQuery request, CancellationToken cancellationToken)
         {
-            var totalCount = await eventRepository.GetCountAsync();
+            var query = new EventQuery(_dbContext.Events);
```

Comment:
@KKDinev20 do not use the new keyword 
just pass IEvnetQuery for in the constructor of the QueryHandler 
and leave the DI container do the magic for you